### PR TITLE
Bump version to 2021

### PR DESCRIPTION
**Pull request by NN---**

### CODE REVIEW

File: src/Renci.SshNet/Properties/CommonAssemblyInfo.cs [hunk 1]
```diff
@@ -5,13 +5,13 @@
 [assembly: AssemblyDescription("SSH.NET is a Secure Shell (SSH) library for .NET, optimized for parallelism.")]
 [assembly: AssemblyCompany("Renci")]
 [assembly: AssemblyProduct("SSH.NET")]
-[assembly: AssemblyCopyright("Copyright � Renci 2010-2017")]
+[assembly: AssemblyCopyright("Copyright � Renci 2010-2021")]
```
Comment (TheRockettek) [inline src/Renci.SshNet/Properties/CommonAssemblyInfo.cs]: Seems the copyright unicode broke here.

Comment (NN---) [reply src/Renci.SshNet/Properties/CommonAssemblyInfo.cs]:
```suggestion
[assembly: AssemblyCopyright("Copyright © Renci 2010-2021")]
```

Comment (NN---) [reply src/Renci.SshNet/Properties/CommonAssemblyInfo.cs]: Fixed. Thanks

**Comment by IgorMilavec** [comment]
Proper metadata will be set as part of the release process.

**Pull request closed without merging**